src/org/ses/android/soap/database/Participant.java

```diff
@@ -48,7 +50,52 @@ public Participant(String CodigoPaciente,
 		this.Sexo = Sexo;
 
 	}
-	
+
+	@Override
+	public int describeContents() {
+		return 0;
+	}
+
+	@Override
+	public void writeToParcel(Parcel dest, int flags) {
+		dest.writeString(CodigoPaciente);
+		dest.writeString(Nombres);
+		dest.writeString(ApellidoPaterno);
+		dest.writeString(ApellidoMaterno);
+		dest.writeInt(CodigoTipoDocumento);
+		dest.writeString(DocumentoIdentidad);
+		dest.writeString(FechaNacimiento);
+		dest.writeInt(Sexo);
+	}
+
+	public static final Parcelable.Creator<Participant> CREATOR =
+			new Parcelable.Creator<Participant>() {
+		public Participant createFromParcel(Parcel in) {
+			return new Participant(in);
+		}
+
+		public Participant[] newArray(int size) {
+			return new Participant[size];
+		}
+	};	
+
+	private Participant(Parcel in) {
+		CodigoPaciente = in.readString();
+		Nombres = in.readString();
+		ApellidoPaterno = in.readString();
+		ApellidoMaterno = in.readString();
+		CodigoTipoDocumento = in.readInt();
+		DocumentoIdentidad = in.readString();
+		FechaNacimiento = in.readString();
+		Sexo = in.readInt();
+	}
+		
+	@Override
+	public String toString() {
+		return "Nombres: " + this.Nombres + "\nApellido Paterno: " + this.ApellidoPaterno +
```

Comment:
Can you put this in `strings.xml` instead of hardcoding it? If you look in `strings.xml` you'll see examples of them using `printf`-like placeholders, e.g. `Nombres: %s %s %s` or something.